### PR TITLE
JBPM-5971: New forms are not visible on the Library.

### PR DESCRIPTION
**Pull request by pefernan**
Adding indexing to forms.

@jsoltes @Rikkola can you take a look?

### CODE REVIEW

File: kie-wb-common-forms/kie-wb-common-forms-editor/kie-wb-common-forms-editor-backend/src/test/java/org/kie/workbench/common/forms/editor/backend/indexing/FormDefinitionIndexerTest.java [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+                                                providersInstance);
+        super.setup();
+    }
+
+    @Test
+    public void testFormsIndexing() throws Exception {
+        List<Path> pathList = new ArrayList<>();
+        for (int i = 0; i < FORMS.length; ++i) {
+            String formFile = FORMS[i];
+            if (formFile.endsWith("frm")) {
```
Comment (jomarko) [inline kie-wb-common-forms/kie-wb-common-forms-editor/kie-wb-common-forms-editor-backend/src/test/java/org/kie/workbench/common/forms/editor/backend/indexing/FormDefinitionIndexerTest.java]: Looking at the content of `FORMS`, is this `if` needed?

Comment (pefernan) [inline kie-wb-common-forms/kie-wb-common-forms-editor/kie-wb-common-forms-editor-backend/src/test/java/org/kie/workbench/common/forms/editor/backend/indexing/FormDefinitionIndexerTest.java]: @jomarko yes, the idea is index the FORMS has the list of forms that has to be indexed on this test.

Comment (jomarko) [inline kie-wb-common-forms/kie-wb-common-forms-editor/kie-wb-common-forms-editor-backend/src/test/java/org/kie/workbench/common/forms/editor/backend/indexing/FormDefinitionIndexerTest.java]: I mean, every item inside of `FORMS` ends with "frm" so this `if` is always true.

**Comment by pefernan** [comment]
@jomarko better now?

**Comment by jomarko** [comment]
@pefernan I know it was minor thing. Sorry for inconvenience. Nothing more to add from my site. Let's wait for @jsoltes.

**Comment by pefernan** [comment]
@jomarko no problem at all, it was an easy change ;)

**Pull request merged**